src/views/BillDisplay/BillDisplay.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {PureComponent} from "react";
+import './styles.scss';
+import { getBill } from "../../utils/requests/BillRequests";
+import loading from "./gif/loading.gif" //temp loader gif
+
+class BillDisplay extends PureComponent {
+
+    state = {   user:   { firstName:  "",
+                        lastName: ""
+                },
+                bills:  [],
+                billsLoaded: false  
+                        
+    };
+
+    /**
+     * tempToken is a temporary token, that should be passed as cookie from login/register
+     * 
+     */
+    tempToken = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJwYXNzd29yZEBQYXNzd29yZDEyMy5jb20iLCJleHAiOjExNTg5NzI4MjE5LCJyb2xlcyI6WyJST0xFX1VTRVIiXX0.qw7RgI7FIIliuzrwg67zpCN2fhRbSGCbVGFmlLRsguN3dp9Qndfn47GncvD9CDwSPZSsOK-2mjYNmEIb5-K45A";
+    
+    /**
+     * @description calls the getBill function to get all bills for a user, then sets the bills states.
+     */
+    fetchBill = async () => {
+                            const bills =  await getBill();
+                            this.setState({...this.state, bills, billsLoaded: true})
+    };
+
+    componentDidMount = async () => {
+                            localStorage.setItem('user', {  firstName: "Bob",
+                                                            lastName: "Smith"});
+                            localStorage.setItem('token', this.tempToken);
+                            await this.fetchBill();
+    };
+
+    billStatusColor = status => {
+        let color = "muted"
+        switch(status){
+            case "OPEN":
+                color = "success";
+                break;
```

Comment:
cleaner way would be to `return "success"` immediately instead of reassigning `color`